mantidproject/vesuvio
# Rearrange package and ammend workflow to run all unit tests

**Description of work:**

This PR:

- Renames the `vesuvio` package `EVSVesuvio`, rearranges the repo to suit.
- Adds a `Mantid.user.properties` file to stop instrument definition download which was causing a segfault. Sets the `MANTIDPROPERTIES` env var to ensure this file is picked up.
- Disables `vesuvio analysis` system tests, as these have never passed.

**To test:**

1. Observe unit tests pass without segfault
2. Check for any instances of `vesuvio` not being renamed `EVSVesuvio`

Resolves #21 and #22
